Makefile

```diff
@@ -195,7 +195,7 @@ test: all
 	$(MAKE) build-addons
 	$(MAKE) cctest
 	$(PYTHON) tools/test.py --mode=release -J \
-		addons doctool inspector known_issues message pseudo-tty parallel sequential
+		doctool inspector known_issues message pseudo-tty parallel sequential addons
```

Comment:
One more thing. CI runs `make test-ci`, not `make test`, so you need to change [line 311](https://github.com/nodejs/node/blob/master/Makefile#L311) too.